### DISCUSSION TITLE
AI-Social 社交媒体

### DISCUSSION THREAD

**Discussion by yjshi2015** [creatorsand/.github#131] [Ideas]
一句话 IDEA：

这是一款基于 AI 的创作工具，支持实时图片生成、音乐生成，同时支持社交互动：用户可以发布、分享和评论内容。代币激励：用户可以通过互动赚取代币，支持代币打赏

代码库：
https://github.com/yjshi2015/ai-social

项目地址：
[ai-social-xpo5.vercel.app](https://ai-social-xpo5.vercel.app/)

我的角色：
产品 & 开发 & 测试

其他角色需求（不包含发起人）：
前端同学

社群昵称：
Gavin

你在社群中的昵称或身份。
Gavin

个人简介：
多年 Web2 后端开发经验，1 年多 Web3 经验，平常主要研究一些公链，最爱 Sui，参加过两场 Sui 生态的黑客松。

产品是给谁用的？
任何人都可以在该平台上体验 AI 创作，并将自己心仪的作品放在区块链中进行永久存储。

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
